packages/components/src/horizontal-bar-list/stats-card.scss

```diff
@@ -24,6 +24,11 @@
 
 	.stats-card--hero {
 		padding-bottom: $card-vertical-space;
+
+		@media (min-width: $break-medium) {
+			min-height: 512px;
+			min-width: 860px;
+		}
```

Comment:
That was added because after moving the title to the left panel, it would render a minimap based on the title column width:

<img width="962" alt="image" src="https://github.com/user-attachments/assets/b94ab5e2-9ebc-480c-a832-1ed86421dac5" />

Ensuring minimum dimensions prevents that. I'm open to suggestions for fixing it that in a better way.